### PR TITLE
Disable forwarding of splunk internal logs and metrics

### PR DESCRIPTION
**Pull request by gsleeman**
Disables forwarding of splunk-internal logs, **except** for:
* `$SPLUNK_HOME/var/log/watchdog/watchdog.log`
* `$SPLUNK_HOME/etc/splunk.version`
* `$SPLUNK_HOME/var/log/splunk/license_usage_summary.log`

Also/minor change:
* fix typo in `containers/forwarder/bin/run.sh`

### CODE REVIEW

File: containers/forwarder/Dockerfile [hunk 1]
```diff
@@ -4,10 +4,10 @@ ADD containers/forwarder/bin/run.sh /
 ADD .splunk-version /
```
Comment (2uasimojo) [inline containers/forwarder/Dockerfile:4]: Drive-by note: since #99 this won't even run unless `.splunk-version[-hash]` has changed.

**Comment by 2uasimojo** [comment]
This needs to be moved over to https://github.com/openshift/splunk-forwarder-images now.

**Comment by gsleeman** [comment]
[Moved to new repo](https://github.com/openshift/splunk-forwarder-images/pull/5)
